app/src/main/java/com/cmput301f20t21/bookfriends/ui/login/CreateAccountViewModel.java

```diff
@@ -19,10 +18,20 @@
 
 
 public class CreateAccountViewModel extends ViewModel {
-    private final IAuthRepository authRepository = AuthRepository.getInstance();
-    private final IUserRepository userRepository = UserRepository.getInstance();
+    private IAuthRepository authRepository = AuthRepository.getInstance();
+    private IUserRepository userRepository = UserRepository.getInstance();
```

Comment:
I don't think we need to initialize here, as we have constructors below?

Reply:
we can remove the ` = xxxRepository.getInstance()`, but keep the `final` keyword as we want to be sure they are only initialized once